example/src/containers/MetaMaskConnector/MetaMaskConnector.tsx

```diff
@@ -20,15 +21,44 @@ export const MetaMaskConnector = () => {
     const installSnap = useCallback(async () => {
        const isInitiated = await installPolkadotSnap();
        if(!isInitiated) {
-           alert("Failed to install snap");
+        dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: false, message: "Please accept snap installation prompt"}})
        } else {
-           addDotAsset().catch(() => alert("Failed to add dot asset"));
+           addDotAsset().catch(() => dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: {isInstalled: false, message: "Failed to add dot asset to metamask"}}));
            dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: true});
        }
     }, [dispatch]);
 
+    const handleClose = (event: React.SyntheticEvent | React.MouseEvent, reason?: string) => {
+        if (reason === 'clickaway') {
+          return;
+        }
+        dispatch({type: MetamaskActions.SET_INSTALLED_STATUS, payload: false})
```

Comment:
you didn't update this payload

Reply:
its just a handleClose function, it sets payload to just false, without message, hides notification when X is pressed, its controled by function in MetaMaskConnector:

const shouldDisplaySnackbar = (): boolean => {
      if (!state.isPolkadotSnapInstalled.isInstalled && state.isPolkadotSnapInstalled.message) return true;
      else return false;
    }